src/collections/components/CollectionProducts/ProductTableItem.tsx

```diff
@@ -0,0 +1,127 @@
+import { ChannelsAvailabilityDropdown } from "@dashboard/components/ChannelsAvailabilityDropdown";
+import { GridTable } from "@dashboard/components/GridTable";
+import Drag from "@dashboard/icons/Drag";
+import { productUrl } from "@dashboard/products/urls";
+import { useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { Box, Button, Checkbox, Skeleton, Text, TrashBinIcon } from "@saleor/macaw-ui-next";
+import React from "react";
+
+import { Product } from "./types";
+
+interface ItemProps {
+  product: Product;
+  isSelected: boolean;
+  draggable: boolean;
+  toggle: (id: string) => void;
+  onProductUnassign: (id: string, event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const ProductTableItem = ({
+  product,
+  isSelected,
+  draggable,
+  toggle,
+  onProductUnassign,
+}: ItemProps) => {
+  const { attributes, listeners, setNodeRef, transform, transition, isSorting } = useSortable({
+    id: product.id,
+    disabled: !draggable,
+  });
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+  const href = product ? productUrl(product.id) : "";
+  const isSaving = product.id.includes("moved_");
+
+  return (
+    <GridTable.Row
+      __height="1px"
```

Comment:
suggestion: `__height="1px"` looks weird, could you use for example `height:auto` or add a comment why 1px?

Reply:
it's, for content inside the row to be full height; the content stretches to the borders, will leave the comment